### DISCUSSION TITLE
Temperaturregler Berker 80440100

### DISCUSSION THREAD

**Discussion by mariushun** [brutella/hkknx-public#320] [Q&A]
Hallo zusammen,

ich versuche schon seit Ewigkeiten meine Berker Temperaturregler in Apple Home zu integrieren. Ich scheitere immer wieder an dem Problem, dass das Gerät voreingestellte Sollwerte für Komfort, Eco, Standby und Frostschutz hat. Diese Sollwerte sind immer aktiv und ich kann mit einer Sollwertverschiebung eine andere Sollwert Temperatur angeben. Ich finde aber nicht heraus, wie ich die im Gerät voreingestellten Sollwerte auslesen und darstellen kann und wie ich die Sollwertverschiebung angeben kann. Darüber hinaus kann ich pro Betriebsmodus auch nur +/-3 Grad Sollwertverschiebung angeben.

Hier mal ein Bild der Adressen des Geräts. Vielleicht kann mir jemand weiterhelfen? Ich verwende einen Hager domovea basic TJA670.

![Bildschirmfoto 2024-05-16 um 12 27 06](https://github.com/brutella/hkknx-public/assets/79144249/d2fb30cc-9325-41c6-9848-d5ca7dd464c6)

Danke im Voraus!

**Comment by chef76** [comment]
Ist das Lesen Flag an der GA des Sollwertes gesetzt?

**Reply by mariushun** [reply]
Also hinter der 24/4/47 stecken aktuell (da Kühlbetrieb) die Sollwerte. Die kann ich auch lesen. Dies ist aber ein DPT213.100 und die Sollwerte aller Betriebsmodi stehen in dem Wert. Ich weiß nicht, wie ich das mit hkknx auslesen soll.

**Reply by chef76** [reply]
Mit einem Skript geht das sicher, da bin ich aber nicht fit genug. Da müsste @brutella helfen.

**Comment by bobobla** [comment]
Gibt es hier schon eine Lösung? Ich struggle mit der gleichen Frage und konnte bisher nichts passendes finden.

**Reply by brutella** [reply]
Dazu gibt es noch keine Lösung, da der Datentyp 213.100 zurzeit noch nicht unterstützt wird.
